dlls/ff/ff_lualib_team.cpp

```diff
@@ -62,8 +86,11 @@ void CFFLuaLib::InitTeam(lua_State* L)
 		class_<CFFTeam, CTeam>("Team")
 			.def("SetAllies",			&CFFTeam::SetEasyAllies)
 			.def("ClearAllies",			&CFFTeam::ClearAllies)
+			.def("GetAllies",			&FFLib::GetAllies)

```

Comment:
A function that takes a class pointer as the first parameter is bound by Luabind the same as a member function of that class

`GlobalFunction(CFFPlayer *obj, int param)` is treated the same as if it were `CFFPlayer::Function(int param)`
From the [Luabind docs](http://www.rasterbar.com/products/luabind/docs.html#binding-classes-to-lua):
> It is also possible to register free functions as member functions. The requirement on the function is that it takes a pointer, const pointer, reference or const reference to the class type as the first parameter. The rest of the parameters are the ones that are visible in Lua, while the object pointer is given as the first parameter.